containers/daap-push-to-catalogue/src/var/task/push_to_catalogue.py

```diff
@@ -0,0 +1,84 @@
+import json
+import logging
+import os
+
+import boto3
+from data_platform_catalogue import CatalogueClient, CatalogueError
+from data_platform_catalogue import DataProductMetadata as omdProductMetadata
+from data_platform_catalogue import TableMetadata
+
+
+def handler(event, context):
+    # can't use daap-python-base as it's python 3.11 and need 3.10 for
+    # data_platform_catalogue, hence we can't use DataPlatformLogger here
+    logger = logging.getLogger()
```

Comment:
can you try using structlog here? (feel free to punt this to a later PR if you want to merge now)

This is the PR where I added it to the base image: https://github.com/ministryofjustice/data-platform/pull/2252

Reply:
yeah why not, i'll add it